src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (reversRobot[value]) {
```

Comment:
p.2 - use correct check for property presence in object. Some properties can be present but still contain falsy values.

Reply:
I don't understand what check you mean.

Reply:
okey, I think I mean it about .hasOwn